make.sh

```diff
@@ -63,8 +63,9 @@ rule() {
       ocb -no-plugin $TARGET.native &&
       cp _build/$TARGET.native goblint
     ;; debug)
```

Comment:
In what sense is this a `debug` build? It's just the bytecode build. I guess that's necessary for earlybird?

Might make sense to call the make target `byte` as well, just like how `native` is consistently called that.

Reply:
`make debug` might be a bit more intuitive for the most common usage of the byte-code file. Anyways I changed the name to `byte` and the documentation accordingly.